frontend/junior-1/html.md

```diff
@@ -50,18 +50,21 @@
         </ul>
     </header>
     ```
-* Зачем нужны метатеги?
-* Что такое data-атрибуты, и привести примеры, когда они полезны?
-* Когда стоит использовать тег `img`, а когда `background-image`?
-* Как стилизовать `svg`? Какие основные атрибуты и CSS-свойства используются для этого?
-* Что такое специальные html-сущности (неразрывный пробел, тире, которое не дефис) и как с ними работать?
+* What are meta tags for?
+* What are data attributes, and give examples of when they are useful?
+* When should the `img` tag be used, and when should `background-image` be used?
+* How to style `svg`? What are the main attributes and CSS properties used for this?
+* What are special HTML entities (non-breaking space, dash, which is not a hyphen) and how to work with them?
 
-### Ресурсы
+### Resources
 
-* [Людоедский интерфейс, Вадим Макеев](https://www.youtube.com/watch?v=ssJsjGZE2sc)
-* [HTML Academy](https://htmlacademy.ru/)
+* [Ludoeedskiy interfeys, Vadim Makeev](https://www.youtube.com/watch?v=ssJsjGZE2sc)
+* [HTML Academy](https://htmlacademy.org/)
 * [HTML (MDN)](https://developer.mozilla.org/ru/docs/Web/HTML)
-* [HTML и HTML5. Описание тегов по основным разделам](https://html5book.ru/html-html5/)
-* [Основы методологии БЭМ](https://ru.bem.info/methodology/quick-start/)
-* [Способы организации CSS-кода](https://habr.com/ru/post/256109/)
-* [SVG: заливка и обводка](http://css.yoksel.ru/svg-fill-and-stroke/)
+<!-- TODO:: find resource on English -->
+* [HTML and HTML5. Description of tags by main sections](https://html5book.ru/html-html5/)
+* [Basics of the BEM methodology](https://en.bem.info/methodology/quick-start/)
+<!-- TODO:: find resource on English -->
+* [Ways to organize CSS code](https://habr.com/ru/post/256109/)
+<!-- TODO:: find resource on English -->
+* [SVG: fill and stroke](http://css.yoksel.ru/svg-fill-and-stroke/)
```

Comment:
There is similar article on [MDN](https://developer.mozilla.org/en-US/docs/Web/SVG/Tutorial/Fills_and_Strokes)